doc/api/errors.markdown

```diff
@@ -6,13 +6,13 @@ Applications running in Node.js will generally experience four categories of
 errors:
 
 - Standard JavaScript errors such as:
-  - [`EvalError`][]: thrown when a call to `eval()` fails.
-  - [`SyntaxError`][]: thrown in response to improper JavaScript language
+  - {EvalError} : thrown when a call to `eval()` fails.
```

Comment:
Sorry @estliberitas , since https://github.com/nodejs/node/pull/5641 fixes the broken links, do we still need to remove them?